rom/dev/src/flow/warm_reset.rs

```diff
@@ -25,11 +25,11 @@ impl WarmResetFlow {
     ///
     /// * `env` - ROM Environment
     #[inline(never)]
-    pub fn run(env: &mut RomEnv) -> CaliptraResult<FirmwareHandoffTable> {
+    pub fn run(_env: &mut RomEnv) -> CaliptraResult<Option<FirmwareHandoffTable>> {
         cprintln!("[warm-reset] ++");
 
         cprintln!("[warm-reset] --");
 
```

Comment:
There will be a conflict here due to https://github.com/chipsalliance/caliptra-sw/pull/516. Suggest keeping the workaround and not returning the FHT (this change).

Reply:
Done.